docs/getting-started/ers-req.asciidoc

```diff
@@ -0,0 +1,44 @@
+[[ers-requirements]]
+= Entity Risk Scoring prerequisites
+
+To use <<entity-risk-scoring, Entity Risk Scoring>>, your role must have certain cluster, index, and {kib} privileges. You also need the https://www.elastic.co/subscriptions[appropriate license].
+
+This page covers the requirements and guidelines for using the Entity Risk Scoring feature, as well as its known limitations.
+
+[discrete]
+== Privileges
+
+To turn on the risk scoring engine, you need the following privileges:
+
+[discrete]
+[width="100%",options="header"]
+|==============================================
+
+| Cluster      | Index | {kib} 
+a| 
+* `manage_index_templates`
+* `manage_transform`
+
+| `all` privilege for `risk-score.risk-score-*`
+
+a| 
+* **All** for the **Saved Objects Management** feature under **Management**
+* **Read** for the **Security** feature 
+
+|==============================================
+
+[discrete]
+== {es} resource guidelines
+
+Follow these guidelines to ensure clusters have adequate memory to handle data volume:
+
+* With 2GB of Java Virtual Machine (JVM) heap memory, the risk scoring engine can safely process around 44 million documents, or 30 days of risk data with an ingest rate of 1000 documents per minute.
+
+* With 1GB of JVM heap, the risk scoring engine can safely process around 20 million documents, or 30 days of risk data with an ingest rate of around 450 documents per minute.
+
+[discrete]
+== Known limitations
+
+* You can only enable the risk scoring engine in a single {kib} space within a cluster.
+
+* The risk scoring engine uses the internal {kib} user to score all hosts and users, and doesn't respect privileges applied to custom users or roles. After you turn on the risk scoring engine for a {kib} space, all alerts in the space will contribute to host and user risk scores.
```

Comment:
I think this refers to a specific default user account name — might want to double-check with Engineering:
```suggestion
* The risk scoring engine uses the internal `kibana` user to score all hosts and users, and doesn't respect privileges applied to custom users or roles. After you turn on the risk scoring engine for a {kib} space, all alerts in the space will contribute to host and user risk scores.
```

Reply:
The task will use the ["kibana internal user"](https://github.com/rylnd/kibana/blob/main/x-pack/plugins/security_solution/server/lib/risk_engine/tasks/risk_scoring_task.ts#L64), however the platform happens to define that at the time. I think the actual user to which that corresponds is nuanced (there's still the `kibana` user, which is now deprecated, and the new `kibana_system` user). Regardless, the important piece is that it will be using the reserved `kibana_system` role and cannot be affected by any custom user roles.

Reply:
Thanks for the extra info, @rylnd! In that case, it's prob OK as originally written, though I wonder if most customers would know what "the internal Kibana user" is — it sounds like under-the-hood developer stuff. Seems like all customers need to know is the engine essentially ignores custom roles.
```suggestion
* The risk scoring engine uses an internal user role to score all hosts and users, and doesn't respect privileges applied to custom users or roles. After you turn on the risk scoring engine for a {kib} space, all alerts in the space will contribute to host and user risk scores.
```

Reply:
Resolved in https://github.com/elastic/security-docs/pull/4162/commits/6662e5229a26821923eefe8d5c8bd9a1b264684b.